packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.6.18+4
+
+* Uses Text.rich to replace RichText so the widget can work with `SelectionArea` when `selectable` set to false.
+* Updates affected tests.
```

Comment:
All changes update affected tests; that's not client-facing, so doesn't go in the CHANGELOG.

Reply:
Done.